src/components/ui/select.tsx

```diff
@@ -0,0 +1,160 @@
+"use client";
```

Comment:
uhm, I'm all for using a UI framework but this looks like you still have to define/config every component by yourself? If yes, what's the point?
I didn't check the the lib in detail. I might not understand what I'm seeing :)

Reply:
See https://ui.shadcn.com/docs

> This is NOT a component library. It's a collection of re-usable components that you can copy and paste into your apps.

> What do you mean by not a component library?

> I mean you do not install it as a dependency. It is not available or distributed via npm.

> Pick the components you need. Copy and paste the code into your project and customize to your needs. The code is yours.

> Use this as a reference to build your own component libraries.

Reply:
ah i see, kinda sad in a way :P

Reply:
Why?

The styling is done via CSS variables. If I add a `components.json` the other components can be added with the CLI.